_docs/using_an_api_like_a_developer/docapis_json_console.md

```diff
@@ -89,11 +89,11 @@ Mashape [provides some sample code](http://docs.mashape.com/javascript) to parse
     <img src="images/jsonpayloadweatherdata.png" alt="JSON payload from weatherdata API logged to console" />
 
     Note that Chrome tells you whether each expandable section is an object or an array. Knowing this is critical to accessing the value through JavaScript dot notation.
-
-    {% include random_ad.html %}
-
+    
     The following sections will explain this AJAX code a bit more.
 
+{% include random_ad.html %}
```

Comment:
The ad wasn't rendering. I don't know Liquid or Jekyll, but my guess is that whitespace is significant and was causing problems.